src/PolyhedralGeometry/LinearProgram.jl

```diff
@@ -1,35 +1,26 @@
-@doc Markdown.doc"""
-    LinearProgram(P, c; k = 0, convention = :max)
-
-The linear program on the feasible set `P` (a Polyhedron) with
-respect to the function x ↦ dot(c,x)+k.
-
-"""
-struct LinearProgram
-   feasible_region::Polyhedron
-   polymake_lp::Polymake.BigObject
-   convention::Symbol
-end
-
-function LinearProgram(Q::Polyhedron, objective::AbstractVector; k = 0, convention = :max)
+function LinearProgram{T}(Q::Polyhedron, objective::AbstractVector; k = 0, convention = :max) where T<:scalar_types
```

Comment:
```suggestion
function LinearProgram{T}(Q::Polyhedron{T}, objective::AbstractVector; k = 0, convention = :max) where T<:scalar_types
```
I don't think we want to allow mixing the types here (or doing an automatic conversion of the polyhedron).